pkg/sql/schemachanger/scdecomp/decomp.go

```diff
@@ -81,12 +80,14 @@ func WalkDescriptor(
 
 func (w *walkCtx) walkRoot() {
 	// Common elements.
-	w.ev(scpb.Status_PUBLIC, &scpb.Namespace{
-		DatabaseID:   w.desc.GetParentID(),
-		SchemaID:     w.desc.GetParentSchemaID(),
-		DescriptorID: w.desc.GetID(),
-		Name:         w.desc.GetName(),
-	})
+	if !w.desc.SkipNamespace() {
```

Comment:
1. IIRC, this `SkipNamespace` is added for functions because we don't want to create rows in the `system.namespace` table for functions, right? (bc we allow functions to have the same name as tables)

2. Do we create multiple rows in `system.descriptor` table for function overloads (i.e. functions with same table but different input arguments)?

Reply:
1. correct.
2. yes, each function overload has its own descriptor.